index.bs

```diff
@@ -109,6 +113,23 @@ and taking the value of the cumulative layout shift score at that time.
 
 This strategy is illustrated in the usage example.
 
+Source attribution {#source-attribution}
+----------------------------------------
+
+<em>This section is non-normative.</em>
+
+In addition to the layout shift value, the API reports a sampling of up to five
+DOM elements whose layout shifts most substantially contributed to the layout
+shit value for an animation frame.
+
+It is possible that the true "root cause" of the instability will be one or more
+steps removed from the DOM elements that experience a layout shift. For example,
+a newly inserted element might cause a shift in content below it. The
+<a href="#dom-layoutshift-sources">sources</a>
+attribute will report only shifted elements, and not the inserted element.
+Nevertheless, we expect this reporting to be of significant value to developers
+who are attempting to diagnose an occurrence of layout instability.
+
```

Comment:
Could we add a note explaining why we're not exposing those 'root causes' instead?